configuration/salt.md

```diff
@@ -321,91 +343,88 @@ domains this way.
         - default:
           - service5
 
-This enables, disables, or sets to default, the services as in qvm-service.
-
-### qvm.running
+This enables, disables, or sets to default, services as in `qvm-service`.
 
+### `qvm.running`
 
-Ensures the domain is running:
+Ensures the specified domain is running:
 
     domain is running:
       qvm.running:
         - name: salt-test4
 
-## qubes pillar module
+## The `qubes` Pillar Module
 
-Additional pillar data is available to ease targeting configuration (for
-example all the templates). List here may be subject to changes in future
-releases.
+Additional pillar data is available to ease targeting configurations (for
+example all templates). 
+***Note*** List here may be subject to changes in future releases.
 
-### qubes:type
+### `qubes:type`
 
 VM type. Possible values:
 
- - `admin` - administration domain (`dom0`)
+ - `admin` - Administration domain (`dom0`)
  - `template` - Template VM
  - `standalone` - Standalone VM
- - `app` - template based AppVM
+ - `app` - Template based AppVM
 
-### qubes:template
+### `qubes:template`
 
-Template name on which given VM is based (if any).
+Template name on which a given VM is based (if any).
 
-### qubes:netvm
+### `qubes:netvm`
 
 VM which provides network to the given VM
 
 ## Debugging
-The output for each vm is logged in `/var/log/qubes/mgmt-VM_NAME.log`.
 
-If the log does not contain useful information, you can stop `qubesctl` by 
-pressing `ctrl+z`.
+The output for each VM is logged in `/var/log/qubes/mgmt-VM_NAME.log`.
 
-You need to:
-1. run `sudo qubesctl --skip-dom0 --target=VM_NAME state.highstate`
-2. When your vm is being started (yellow) press Ctrl-Z on qubesctl.
-3. Open terminal in disp-mgmt-VM_NAME.
-4. Look at /etc/qubes-rpc/qubes.SaltLinuxVM - this is what is 
-   executed in the management vm.
-5. Get the last two lines:
+If the log does not contain useful information:
+1. Stop `qubesctl` using Ctrl-z
```

Comment:
Ok, so I misunderstood this.  It originally said, "If the log does not contain useful information, you can stop `qubesctl` by 
-pressing `ctrl+z`." and I wanted to make sure that it was just a note rather than a step.  I left it in there just in case.  I'll delete this and move the steps down one.